### ISSUE TITLE
Calendar: accessibility

### ISSUE THREAD

**Issue by rbenyona** [primefaces/primefaces#1177]
The calendar component does not have W3C Web content Accessibility support

**Comment by dariorios** [comment]
I agree with that. Is a completely necessary feature in accesibility.

**Comment by kukel** [comment]
This calendar component from https://blog.jqueryui.com/2014/06/jquery-ui-1-11-0/ is used. http://plugins.jquery.com/ui.datepicker/ Check if the newer ones contain what you need. If so, please state here to upgrade. If not, please file issues at that component

**Comment by kukel** [comment]
4.3 on http://wiki.jqueryui.com/w/page/12137778/Datepicker and https://lists.w3.org/Archives/Public/public-aria/2016Jan/0166.html. So most of it is out of PF hands

**Comment by shiva1248** [comment]
Hi,

Any update when would primefaces include W3C Web content Accessibility support for calendar component ?

Any timelines for this one ?

**Comment by christophs78** [comment]
@tandraschko : We may try to backport https://github.com/primefaces/primeng/issues/4695, https://github.com/primefaces/primeng/commit/ea9596bf6831270a708633be0a40aadecf92ff15 from PrimeNG for DatePicker.  (DatePicker is the replacement for Calendar)
It´s a bit sad PrimeTek only added this for PrimeNG, PrimeReact and PrimeVue. Not much love for PrimeFaces.

**Comment by christophs78** [comment]
https://github.com/primefaces/primefaces/issues/4685

**Comment by melloware** [comment]
Closing this ticket as Jquery UI is where the Calendar code comes from so not much PF can fix, i would open a ticket on Jquery UI page..
